inc/namespace.php

```diff
@@ -50,3 +56,109 @@ function load_publication_checklist() {
 
 	require_once Altis\ROOT_DIR . '/vendor/humanmade/publication-checklist/plugin.php';
 }
+
+/**
+ * Load Yoast Duplicate Posts, if enabled.
+ */
+function load_duplicate_posts() {
+	$config = Altis\get_config()['modules']['workflow']['clone-republish'] ?? null;
+
+	// Bail if Clone & Republish is disabled.
+	if ( ! $config ) {
+		return;
+	}
+
+	// Load the main plugin file.
+	require_once Altis\ROOT_DIR . '/vendor/yoast/duplicate-post/duplicate-post.php';
+}
+
+/**
+ * Remove the Duplicate Post settings page.
+ */
+function remove_duplicate_post_admin_page() {
+	remove_submenu_page( 'options-general.php', 'duplicatepost' );
+}
+
+/**
+ * Override the Duplicate Post supported post types if the config option defines new or different post types than the defautls.
+ *
+ * Default supported post types are all public post types.
+ *
+ * Note: The configured option will _replace_ any enabled post types by default.
+ *
+ * @param array $enabled_post_types The post types that are supported by Duplicate Post.
+ * @return array The filtered array of supported post types.
+ */
+function set_enabled_post_types( array $enabled_post_types ) : array {
+	$post_types = Altis\get_config()['modules']['workflow']['clone-republish']['post-types'] ?? null;
+
+	if ( ! $post_types ) {
+		return $enabled_post_types;
```

Comment:
Sorry for the runaround here I'm just trying to get this straight in my head - if this is the only place we're filtering the default post types then we probably want to return all public post types here, then the helper function can just return the result of applying the filter to an empty array.

Does this work currently for a custom public post type? Quick way to check is to add a muplugin like:

```php
add_action( 'init', function () {
    register_extended_post_type( 'project' );
    register_taxonomy_for_object_type( 'category', 'project' );
    register_taxonomy_for_object_type( 'post_tag', 'project' );
} );
```

Reply:
No, you're good. I cleaned up the logic a bit, refactored the `get_duplicate_post_types` function to be exclusively a helper that returns whatever was saved in the `duplicate_post_enabled_post_types` filter and added the `get_post_types` check here, so we can use that _instead of_ whatever is coming from Yoast internally.

Tested and it currently works so that the options are either _all_ public post types or whatever post types are defined in the config.